gbus/saga/glue.go

```diff
@@ -190,7 +190,9 @@ func (imsm *Glue) invokeSagaInstance(instance *Instance, invocation gbus.Invocat
 		inboundMsg:          message,
 		sagaID:              instance.ID,
 		ctx:                 invocation.Ctx(),
-		invokingService:     imsm.svcName}
+		invokingService:     imsm.svcName,
+		logger:              imsm.log().WithField("saga_id", instance.ID),
```

Comment:
I would add the type of the saga  and maybe the type/message fqn that invokes the saga instance as fields as well